File: calc-java/calc.java
import java.util.Scanner;

public class calc {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    System.out.println("Enter the first number: ");
    double num1 = scanner.nextDouble();
    System.out.println("Enter the second number: ");
    double num2 = scanner.nextDouble();
    System.out.println("Enter the operation (+, -, *, /): ");
    String operation = scanner.next();
    double result = 0;
    switch (operation) {
      case "+":
        result = num1 + num2;
        break;
      case "-":
        result = num1 - num2;
        break;
      case "*":
        result = num1 * num2;
        break;
      case "/":
        result = num1 / num2;
        break;
      default:
        System.out.println("Invalid operator");
        break;
    }
    System.out.println("Result: " + result);
    scanner.close();
  }
}

File: dart-calc/calc.dart
import 'dart:io';

void main() {
  stdout.write('Enter first number: ');
  int num1 = int.parse(stdin.readLineSync()!);

  stdout.write('Enter second number: ');
  int num2 = int.parse(stdin.readLineSync()!);

  stdout.write('Enter operator (+, -, *, /): ');
  String operator = stdin.readLineSync()!;

  int result;
  switch (operator) {
    case '+':
      result = num1 + num2;
      break;
    case '-':
      result = num1 - num2;
      break;
    case '*':
      result = num1 * num2;
      break;
    case '/':
      result = num1 ~/ num2;
      break;
    default:
      stdout.write('Invalid operator');
      return;
  }

  stdout.write('Result: $result');
}
